sklearn/decomposition/sparse_pca.py

```diff
@@ -66,6 +66,13 @@ class SparsePCA(BaseEstimator, TransformerMixin):
         If None, the random number generator is the RandomState instance used
         by `np.random`.
 
+    normalize_components : boolean
```

Comment:
I would show in the docstring that `normalize_components` is optional and which is its default.

https://github.com/scikit-learn/scikit-learn/blob/bcd6ff387e3b273fb5edf2fb0a12497655eb2df9/sklearn/ensemble/bagging.py#L477